### DISCUSSION TITLE
Block on Ping Test inside Github Workflow?

### DISCUSSION THREAD

**Discussion by siegfried01** [community/community#36274] [Actions]
I am cross posting from [https://learn.microsoft.com/en-us/answers/questions/1035694/how-to-make-github-workflow-wait-for-bicep-deploym.html](url) because it has been a while with no resolution and maybe this is a more appropriate forum.

As described in the above link, it appears my github action "azure/arm-deploy@v1" to deploy my web site is asynchronous and the subsequent code in my github workflow is failing because the web site does not exist yet .. or maybe it does exist and the DNS table has not been updated yet...

Since posting the above link, I've been thinking the solution would be to repeatedly ping the web site inside a loop until the ping succeeds.

What is the easiest way to do this? Is there a github action to help me out? Maybe a little script? 

Thanks

Siegfried

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @siegfried01 ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Reply by siegfried01** [reply]
Thanks!
